javascript-big-practice/src/scripts/login.js

```diff
@@ -0,0 +1,38 @@
+const form = document.getElementById("login");
+let data;
+
+form.addEventListener("submit", async (e) => {
+  e.preventDefault();
+  const input = {
+    email: document.getElementById("email").value,
+    password: document.getElementById("pass").value,
+  };
+
+  const res = await fetch("http://localhost:3000/users", {
+    method: "GET",
+    mode: "cors",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+
+  if (!res.ok) {
+    alert(`Error ${res.status}: ${res.statusText}!`);
+  }
+
+  data = await res.json();
+  if (validateUser(data, input)) {
```

Comment:
You should request api with param(email, password to check if the user is valid instead of fetching all users and checking mapping user from local.